symExec.py

```diff
@@ -1510,19 +1509,16 @@ def sym_exec_ins(start, instr, stack, mem, global_state, path_conditions_and_var
             else:
                 log.debug("temp: " + str(stored_address))
                 temp = ((stored_address + 31) / 32) + 1
-                if isReal(current_miu_i):
-                    current_miu_i = BitVecVal(current_miu_i, 256)
                 log.debug("current_miu_i: " + str(current_miu_i))
                 expression = current_miu_i < temp
                 log.debug("Expression: " + str(expression))
                 solver.push()
                 solver.add(expression)
                 if solver.check() != unsat:
                     # this means that it is possibly that current_miu_i < temp
-                    if expression:
-                        current_miu_i = temp
-                    else:
-                        current_miu_i = If(expression,temp,current_miu_i)
+                    current_miu_i = temp
```

Comment:
Hey, I dont think you can assign current_miu_i to temp here all the time. You still need the check in the expression